radio/src/gui/colorlcd/listbox.cpp

```diff
@@ -205,10 +205,14 @@ void ListBase::checkEvents(void)
 
   if (isLongPress()) {
     if (longPressHandler) {
-      setSelected(yDown / lineHeight);
+      if(selectionType == LISTBOX_SINGLE_SELECT)
+        setSelected(yDown / lineHeight);
+      else
+        setFocusLine(yDown / lineHeight);
       longPressHandler(0);
       killAllEvents();
```

Comment:
I didn't look into too much detail but expected this killAllEvents to stop onTouchEnd from happening, but it doesn't. The waslongpress was added to ignore the touchend from causing a normal touch as well as long. Hope this is okay.

Reply:
Yeah, the naming is unfortuate... it should really be killAllKeyEvents :-/ ... but is understandable as until touch, key inputs were the only events interested in.

Reply:
I think `touchState.event = TE_NONE;` kills the touch event, but given this manipulation is done only one other place in main code, I think I would prefer there be an actual function that does this, as I see this more as fiddling with a libopenui touch state variable. Regardless, this should do for now.